packages/opencensus-web-instrumentation-zone/test/test-interaction-tracker.ts

```diff
@@ -339,11 +388,22 @@ describe('InteractionTracker', () => {
   }
 
   function doHTTPRequest() {
+    // Set the ocTraceOrigins value so the `traceparent` context header is set.
+    (window as WindowWithOcwGlobals).ocTraceHeaderHostRegex = /.*/;
```

Comment:
Could you do a test case of an XHR to an origin that does not match the regex to demonstrate both cases of that `if`?

Reply:
Done.